python/setup.cfg

```diff
@@ -0,0 +1,59 @@
+#   LICENSE
+# Copyright (c) 2021 Genome Research Ltd.
+# Author: Cancer Genome Project cgphelp@sanger.ac.uk
+#
+#
+# This file is part of CASM-Smart-Phase.
+#
+# CASM-Smart-Phase is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation; either version 3 of the License, or (at your option) any
+# later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+#    1. The usage of a range of years within a copyright statement contained within
+#    this distribution should be interpreted as being equivalent to a list of years
+#    including the first and last year specified and all consecutive years between
+#    them. For example, a copyright statement that reads ‘Copyright (c) 2005, 2007-
+#    2009, 2011-2012’ should be interpreted as being identical to a statement that
+#    reads ‘Copyright (c) 2005, 2007, 2008, 2009, 2011, 2012’ and a copyright
+#    statement that reads ‘Copyright (c) 2005-2012’ should be interpreted as being
+#    identical to a statement that reads ‘Copyright (c) 2005, 2006, 2007, 2008,
+#    2009, 2010, 2011, 2012’."
+#
+#
+
+[metadata]
+name = casmsmartphase
+version = 0.0.1
+author = David Jones
+author_email = cgpit@sanger.ac.uk
+description = Tools associated with Smart-phase MNV phasing
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/cancerit/CASM-Smart-Phase/
+download_url = https://github.com/cancerit/CASM-Smart-Phase/
+classifiers =
+  Programming Language :: Python :: 3
+  License :: OSI Approved :: AGPL3 License
+  Operating System :: OS Independent
+
+[options]
+install_requires =
+  vcfpy
+  pysam
+packages = casmsmartphase
+python_requires = >=3.6
```

Comment:
This needs confirming to be true.  Better to say it's 3.7 and tested known to work than include 3.6 (which reaches end of life in December)